packages/docusaurus-theme-classic/src/theme/Layout/styles.css

```diff
@@ -19,3 +19,9 @@ body {
 .main-wrapper {
   flex: 1 0 auto;
 }
+
+/* Docusaurus-specific utility classes */
+
+.docusaurus-mt-lg {
```

Comment:
To avoid creating unnecessary classes in css.modules files and to reduce code duplication, I think it makes sense to create own utility class for `margin-top`.

Reply:
I find it a bit confusing personally, as the classname let think about a design system with a naming convention, but in practice there is only one class

If 3rem is a value we plan to use everywhere, it should be set in the theme design system (ie infima?). Otherwise I think this is a case where duplication is better than the wrong abstraction (https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction)

Reply:
Yes, I agree with you, but it's hard to adding this margin inside Infima's current codebase. This is something in between[ `md` and `lg`](https://github.com/facebookincubator/infima/blob/9e0a10b921bde64e1d92740e2521ed20fc590fbe/packages/core/styles/layout/spacing.pcss#L9-L10), and I don't know what to choose the right name for this value, so I decided to create utility class directly in the Docusaurus code.

Reply:
If the only purpose is deduplication, wonder if something `.docusaurus-mt-3rem` wouldn't be clearer

No strong opinions so I'll let you merge if you think it's good enough

Reply:
Yes, that's right. I did not use absolute units in the class name in order to avoid changing this class name in the components themselves if we change the value in the future from `3rem` to `3.5rem` as example.